### ISSUE TITLE
icpc command not found, Installation and ussage step

### ISSUE THREAD

**Issue by sidhantwadhwani** [xgfs/verse#11]
Dear Team,

I have been trying to use this repository from mu Ubuntu system, when I run this command "make CXX=icpc", i get the error "make icpc command not found \n makefile:25: recipe for target 'verse-library' failed". I have looked for this error and tried multiple ways to resolve this issue, but some how nothing seems to work. It would be great if yo can provide me a way to resolve this issue, or to reach the binary file which is generated by these step, so that I can use that binary file in the python wrapper class and execute the code.

Kindly provide any help feedback on this.

Regards,
Sidhant

**Comment by xgfs** [comment]
Hi,

icpc is Intel's c++ compiler.
https://software.intel.com/content/www/us/en/develop/tools/compilers/c-compilers.html

You can also use gcc, which will work just fine.

**Comment by sidhantwadhwani** [comment]
Hello, 

I already have the c++ compilers (gcc and g++ both) installed on my system. Also I tried running the file example.sh (instead of running the three command given under "Installation and Usage"). This example.sh file runs successfully(i.e does not give any error), but it does not generate the '.bin' file anywhere. As per my understanding(correct me if i am wrong), after running the example.sh file, it should generate the binary file, which we can use with the python wrapper for generating the embedding.

**Comment by xgfs** [comment]
No, you are not correct. wrapper.py should work after you run `make` in the src folder. You need to input the correct cpath parameter to the wrapper.
